ducktape/tests/reporter.py

```diff
@@ -36,9 +36,11 @@ class SimpleReporter(TestReporter):
     def header_string(self):
         """Header lines of the report"""
         header_lines = [
+            "=========================================================================",
             "Test run with session_id " + self.results.session_context.session_id,
             self.pass_fail(self.results.get_aggregate_success()),
-            "---------------------------------------------------------------------"
+            "==========================="
```

Comment:
Ok, definitely a nit here, but I think I will go crazy if these lines demarcating the report sections are not the same length :)